plugin-flex-ts-template-v2/package.json

```diff
@@ -30,6 +30,8 @@
     "prop-types": "^15.7.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-icons": "^4.8.0",
+    "react-select": "^5.5.0",
```

Comment:
Instead of bringing in react-icons, can we bring in a single icon to the `icons` dir or use one of [the Flex UI icons](https://assets.flex.twilio.com/docs/releases/flex-ui/2.2.0/theming/icons/flex-icons/)?

Can you remove react-select? It is no longer used, probably got added as part of merge fun.